docs/samples/collection_job_driver_config.yaml

```diff
@@ -75,3 +75,6 @@ worker_lease_clock_skew_allowance_secs: 60
 # Number of times to attempt processing an collection job before abandoning it.
 # (required)
 maximum_attempts_before_failure: 10
+
+# Number of sharded database records per batch aggregation.
```

Comment:
It might be worth adding a comment to the various files saying that this setting needs to match across the configuration of different components.